modules/http/src/main/java/org/mule/module/http/internal/request/DefaultHttpRequester.java

```diff
@@ -87,8 +86,10 @@
     static final String WORKSTATION_DEBUG = "Workstation";
     static final String AUTHENTICATION_TYPE_DEBUG = "Authentication Type";
     static final String QUERY_PARAMS_DEBUG = "Query Params";
-    private static final String REMOTELY_CLOSED = "Remotely closed";
-    private static final int DEFAULT_RETRY_ATTEMPTS = 3;
+    static String RETRY_ATTEMPTS_PROPERTY = SYSTEM_PROPERTY_PREFIX + "http.client.requestAttempts";
```

Comment:
-> maxRetry?

Reply:
client or requester? Are there any other system properties already to consider consistency?

Reply:
Well, I could see the http.client.headerSectionSize system property.